.github/workflows/release.yml

```diff
@@ -113,7 +113,13 @@ jobs:
         run: |
           echo "Importing certificate"
           echo "${{ secrets.MACOS_CERTIFICATE }}" | base64 --decode > certificate.p12
-          security create-keychain -p "${{ secrets.MACOS_CERTIFICATE_PASSWORD }}" build.keychain
+          # Check if the keychain exists
```

Comment:
This must not be necessary or else we might leak some keys from one workflow to another. There must be some state sharing here that causes an issue that is not present in GitHub's runners and thus needs to be fixed.

Reply:
I checked the runner and the keychain already exists on the machine, hence why the error the keychain exists already. What do you propose to do to solve this?

Reply:
I'm fairly certain it exists because of the command that was ran earlier. I don't know how runners work, but it is expected that every time we have a clean environment with no key chains and nothing in them. Maybe previous workflow run created it, but then something must ensure it doesn't exists anymore after it is over or else, as mentioned above, we're leaking keys.

And we need to check the same for Windows. It may not complain, but keys might still be left in there.

Reply:
The cleanup script should wipe this now after each run for macOS; for windows, it stores it in the path where jobs are ran and cleaned up, so should be fine.